src/commands/server/logout.ts

```diff
@@ -0,0 +1,34 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command } from '@oclif/command'
+import { cli } from 'cli-ux'
+
+import { CheServerLoginManager } from '../../api/che-login-manager'
+
+export default class Logout extends Command {
+  static description = 'log out of Eclipse Che server'
+
+  async run() {
+    const loginManager = await CheServerLoginManager.getInstance(this.config.configDir)
+    const currentLogin = loginManager.getCurrentLoginInfo()
+
+    const cheApiEndpoint = currentLogin.cheApiEndpoint
+    const username = currentLogin.username
+    if (!cheApiEndpoint || !username) {
+      cli.info('Not currently logged in')
+      return
+    }
+
+    loginManager.deleteLoginContext(cheApiEndpoint, username)
```

Comment:
We should also invoke `logout` api on che-server

Reply:
We use Keycloak for securing Che (so login/logout is done via Keycloak too). However there is no way to revoke single token, but whole session. Do we need that?